kohttp-jackson/build.gradle.kts

```diff
@@ -11,7 +11,7 @@ dependencies {
     testImplementation("org.mock-server:mockserver-netty:5.5.4")
     testImplementation("org.mock-server:mockserver-client-java:5.5.4")
 
-    testImplementation("org.assertj:assertj-core:3.12.2")
+    testImplementation("org.assertj:assertj-core:3.15.0")
```

Comment:
The initial reason why I bumped the version was because I thought that a function of AssertJ was not available. But the problem was that my editor was not correctly importing the static method. So I could safely put the version back to 3.12.2 but I thought it would be good to bumped to the latest anyway. I thought the latest version was 3.15.0 but it looks like I didn't validated it at maven central, sorry! I have bumped it to 3.16.1